crates/matrix-sdk/src/test_utils/events.rs

```diff
@@ -213,6 +219,17 @@ impl EventFactory {
         builder
     }
 
+    /// Create a replacement message for an existing one given its id.
+    pub fn replacement_msg(
+        &self,
+        new_content: impl Into<String>,
+        original_event_id: &EventId,
+    ) -> EventBuilder<RoomMessageEventContent> {
+        let msg_content = RoomMessageEventContent::text_plain(new_content.into());
+        let builder = self.event(msg_content.clone());
+        builder.replaces(original_event_id, msg_content)
+    }
```

Comment:
Is this function needed? It's just a shortcut to other functions which can be used directly, and it would avoid adding too much abstraction in the tests themselves.

Reply:
It's a helper, so I expect it to contain boilerplate to avoid repetitive code. Should I remove the `redaction` fn above too?